src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/SdkContentTests/MsftToSbSdkFiles.diff

```diff
@@ -14,9 +14,9 @@ index ------------
  ./packs/Microsoft.AspNetCore.App.Ref/
  ./packs/Microsoft.AspNetCore.App.Ref/x.y.z/
 @@ ------------ @@
- ./packs/Microsoft.AspNetCore.App.Ref/x.y.z/ref/netx.y/System.Security.Cryptography.Xml.xml
- ./packs/Microsoft.AspNetCore.App.Ref/x.y.z/ref/netx.y/System.Threading.RateLimiting.dll
- ./packs/Microsoft.AspNetCore.App.Ref/x.y.z/ref/netx.y/System.Threading.RateLimiting.xml
+ ./packs/Microsoft.AspNetCore.App.Ref/x.y.z/ref/net10.0/System.Security.Cryptography.Xml.xml
```

Comment:
This should not be 10.0. I'm assuming that the version regex/matcher needs to be updated to consume multi-digit major versions

Reply:
Fixed in https://github.com/dotnet/sdk/pull/43964/commits/3cc7dbf37dd33de362e94131baa8307b545570e8